src/modules/moduleRegistry.tsx

```diff
@@ -1,14 +1,18 @@
 import { FC } from "react";
 import { PageModulesCollectionFragment } from "@/lib/contentful/fragments/PageModulesCollectionFragment";
 import { ResultOf, TadaDocumentNode } from "gql.tada";
+
+// Queries
 import { GetModuleExpertsOverflowById } from "@/lib/contentful/query/GetModuleExpertsOverflowById";
 import { GetModuleCustomerStoriesOverflowById } from "@/lib/contentful/query/GetModuleCustomerStoriesOverflowById";
-import { ModuleChapterGroup } from "./ModuleChapterGroup";
 import { GetModuleChapterGroupById } from "@/lib/contentful/query/GetModuleChapterGroupById";
+import { GetModuleInsightsBentoById } from "@/lib/contentful/query/GetModuleInsightsBentoById";
 
 // Modules
 import { ModuleExpertsOverflow } from "./ExpertsOverflow";
 import { ModuleCustomerStoriesCarousel } from "./ModuleCustomerStoriesCarousel";
+import { ModuleChapterGroup } from "./ModuleChapterGroup";
```

Comment:
Ftr, I'd be open to just putting the components and queries right next to each other (or in the same module) to kind of clear up this awkwardness of linking them here... but don't know if it's the _best_ idea.

Reply:
do you mean something like:
`import { ModuleInsightsBento, GetModuleInsightsBentoById } from "./ModuleInsightsBento";`?